components/CheckStatusResponses/CheckStatusShippingCanadaPost.tsx

```diff
@@ -13,25 +13,28 @@ export const CheckStatusShippingCanadaPost: FC<
 
   return (
     <>
-      <p data-testid="shipped-canada-post">
-        {t('shipped-canada-post.printed-and-mailing')}
-      </p>
+      <h2 data-testid="shipped-canada-post" className="h2 text-blue-normal">
+        {t('status-check-passport-printed')}
+      </h2>
+      <span>{t('shipped-canada-post.mailing')}</span>
```

Comment:
Should be wrapped with a `p` tag.

```tsx
<p>
        {t('shipped-canada-post.mailing')}
        {trackingNumber && (
          <>
            {t('status-check-tracking.number')} <b>{trackingNumber}</b>.<br />
            <ExternalLink
              href={t('status-check-tracking.link.canada-post', {
                trackingNumber,
              })}
            >
              {t('status-check-tracking.link.text')}
            </ExternalLink>
          </>
        )}
      </p>
```

`\n` shouldn't be used for the link text

Reply:
Fair enough, I was originally using `<br />` to force the new line but figured the `whitespace-pre-line` was a cleaner alternative